### PR TITLE
fix: broken `README.md` link

### PR DESCRIPTION
**Pull request by alvarobartt**
# Description

Just a minor fix on a broken link in the `README.md`.

**Type of change**

- [X] Documentation update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ You can choose one of many amazing frameworks to train your model, like [transfo
 Check out our [deep dives](https://docs.argilla.io/en/latest/guides/guides.html) and our [tutorials](https://docs.argilla.io/en/latest/tutorials/tutorials.html) on how Argilla integrates with these frameworks.
 
 
-If you want to train a Hugging Face transformer or spaCy NER model, we provide a neat shortcut to [prepare your dataset for training](https://docs.argilla.io/en/latest/guides/features/datasets.html#Prepare-dataset-for-training).
+If you want to train a Hugging Face transformer or spaCy NER model, we provide a neat shortcut to [prepare your dataset for training](https://docs.argilla.io/en/latest/guides/log_load_and_prepare_data.html?highlight=prepare%20dataset#Prepare-dataset-for-training).
```
Comment (tomaarsen) [inline README.md]:
```suggestion
If you want to train a Hugging Face transformer or spaCy NER model, we provide a neat shortcut to [prepare your dataset for training](https://docs.argilla.io/en/latest/guides/log_load_and_prepare_data.html#Prepare-dataset-for-training).
```
The highlighting is a bit overkill I think - it's also highlighting "dataset" everywhere.

**Review by tomaarsen — CHANGES_REQUESTED** [review]
Thanks for spotting this! One small nit

File: README.md (hunk 1 above)
Comment (alvarobartt) [reply README.md]: Sure, thanks!

**Review by tomaarsen — APPROVED** [review]
Looks good to me now.

@frascuchon should this PR point to `main` instead?

**Comment by dvsrepo** [comment]
Thanks @alvarobartt 🙏🙏
I'd like this to point develop as it is the default branch we show to newcomers .

**Comment by tomaarsen** [comment]
Alright, perfect!

**Pull request merged**